src/components/ebay-listbox-button/index.js

```diff
@@ -58,6 +58,9 @@ module.exports = require('marko-widgets').defineComponent({
     handleCollapse() {
         this.emit('listbox-collapse');
     },
+    handleMouseDown() {
```

Comment:
This will only handle when its selected by a mouse click.  Don't we need the same behavior for keyboard?  Why not just put this in handleListboxChange?

Reply:
All the behavior for keyboard is already handled. We only needed to handle the click from the mouse.